src/player/inventory.rs

```diff
@@ -0,0 +1,408 @@
+use std::os::linux::raw::stat;
```

Comment:
Is this used? It fails on mac and win, and I can't find any references to it

Reply:
No, that is probably an error due to autoimport. I have probably pressed enter too early when writing `state`...
I am doing a cleanup with clippy right now and will be removing all the unused imports, like this one